core/templates/pages/exploration-player-page/services/exploration-engine.service.spec.ts

```diff
@@ -511,6 +512,51 @@ describe('Exploration engine service ', () => {
       expect(isAnswerCorrect).toBe(true);
     });
 
+    // it('should check for misspellings if the answer ' +
```

Comment:
Why is this commented out? Please check the PR properly before asking for a review.

Reply:
So sorry! Fixed.